CONTRIBUTING.md

```diff
@@ -7,7 +7,10 @@
 
 # Prerequisites:
 
-Install VS 2019 (Professional or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
+- Install VS 2019 (Professional or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
+- Install the **ASP.NET and web development** and **.NET desktop development** workloads in VisualStudio
+- Install the latest version of **.NET Core SDK** for your specific platform. (https://dotnet.microsoft.com/download)
+- Install Visual Studio Code
```

Comment:
How is Visual Studio Code required?  I don't have it installed.

Reply:
VS COde is not required, sorry I should put an optional in front of that.